.github/workflows/ci.yaml

```diff
@@ -73,6 +73,7 @@ jobs:
       - name: Install just
```

Comment:
If you want to get a specific version of just, you can do `curl --proto '=https' --tlsv1.2 -sSf https://just.systems/install.sh | bash -s -- --to DEST --tag VERSION`

Reply:
Does that work on windows?

Reply:
Uhhhhhhh, I don't actually know 😅

Reply:
Yeah, no, it doesn't. :|

Reply:
I've opened this issue to explain what the actual problems are: https://github.com/casey/just/issues/943. I.e. I'm not at all interested in getting a specific `just` version, the newest would be just fine. I just want a way that works on github CI on all platforms.